src/jetstream/cnsi.go

```diff
@@ -15,6 +15,7 @@ import (
 	"crypto/sha1"
 	"encoding/base64"
 
+	"github.com/cloudfoundry-incubator/stratos/src/jetstream/plugins/userfavorites/userfavoritesendpoints"
```

Comment:
Would be good to get the nod from Neil on this pattern. Apart from a clunky `unregister` hook in all plugins I can't think of a better way. Functionality wise everything looks ok